app/src/test/java/uk/gov/functional/FunctionalTest.java

```diff
@@ -73,7 +73,7 @@ private void send(List<String> payload) {
         Response response = jerseyClient.target("http://localhost:4568/load")
                 .request(MediaType.APPLICATION_JSON_TYPE)
                 .post(Entity.json(String.join("\n", payload)));
-        if (response.getStatus() != 200) {
+        if (response.getStatus() != 204) {
```

Comment:
why did this change? (and should the test maybe just ensure it's in the 2xx range?)